index/index_model.go

```diff
@@ -151,6 +155,17 @@ type SpecIndexConfig struct {
 	uri []string
 }
 
+// SetTheoreticalRoot sets the spec file paths to a theoretical file path
```

Comment:
Can you just add a little more here to explain what this means for folks who don't know why we added this?